src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -137,7 +137,7 @@ public function getTypes($class, $property, array $context = array())
             return;
         }
 
-        if (!in_array($prefix, ReflectionExtractor::$arrayMutatorPrefixes)) {
+        if (!in_array($prefix, ReflectionExtractor::$defaultArrayMutatorPrefixes)) {
```

Comment:
Duly noted. Do you think default values for these instance properties should come from `ReflectionExtractor::$default*Prefix`?